sdk/identity/azure-identity/azure/identity/_credentials/azure_pipelines.py

```diff
@@ -59,10 +44,16 @@ class AzurePipelinesCredential:
     This credential enables authentication in Azure Pipelines using workload identity federation for Azure service
     connections.
 
-    :keyword str service_connection_id: The service connection ID, as found in the querystring's resourceId key.
-        Required.
-    :keyword str tenant_id: ID of the application's Microsoft Entra tenant. Also called its "directory" ID.
-    :keyword str client_id: The client ID of a Microsoft Entra app registration.
+    :keyword str tenant_id: The tenant ID for the service connection. Required.
+    :keyword str client_id: The client ID for the service connection. Required.
+    :keyword str service_connection_id: The service connection ID for the service connection associated with the
```

Comment:
It's worth mentioning where to find this value. See this example in .NET: https://github.com/Azure/azure-sdk-for-net/blob/79e3270e86d86f074c117b2014e58ec7d8820b1c/sdk/identity/Azure.Identity/src/Credentials/AzurePipelinesCredential.cs#L38

Reply:
Should we use an aka link for it? Then we can add screenshots.

Reply:
If I remember correctly, last time when we discussed, we agreed that it was tricky that the service connection id was not on the page but in the url and it was helpful if we could add a screenshot.

Reply:
Can add additional info and screenshots in the TSG, but for the docstring here, I added a bit more info since I felt "found in the querystring's resourceId key" seemed a bit too vague. 

@scottaddie Let me know what you think.

Note, we could also potentially link: https://learn.microsoft.com/azure/devops/pipelines/library/service-endpoints if we think that'd be useful for users.

Reply:
If we want to add link here, use aka link so we reserve the flexibility.

Reply:
Xiang is correct. We had a 1P customer point out that a screenshot would have been useful for locating the service connection ID. I think the TSG is the place to put this.